lib/capistrano/tasks/migrations.rake

```diff
@@ -0,0 +1,123 @@
+namespace :forkcms do
+  namespace :migrations do
+    desc 'Prepare the server for running Fork CMS migrations'
+    task :prepare do
+      on roles(:web) do
+        create_migrations_file
+      end
+    end
+
+    desc 'Execute pending migrations'
+    task :execute do
+      on roles(:web) do
+        # Prepare for migrations if it's not done already
+        Rake::Task["forkcms:migrations:prepare"].invoke()
+
+        # If the current symlink doesn't exist yet, execute the first migration
+        execute_initial_migration if not test "[[ -e #{current_path} ]]"
+
+        # Abort if no migrations are found
+        migration_folders = get_migration_folders
+        if migration_folders.length == 0
+          next
+        end
+
+        # Fetch unprocessed migrations
+        migrations_to_execute = get_migrations_to_execute
+
+        # Abort if no new migrations are found
+        if migrations_to_execute.length == 0
```

Comment:
hier kunt ge ook oneliner van maken, in Ruby zijn ze daar nogal fan van: `next if ...`

Reply:
fe3e4be